desci-server/src/controllers/auth/revokeApiKey.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from 'express';
+
+import { prisma } from '../../client.js';
+import { logger as parentLogger } from '../../logger.js';
+interface RevokeApiKeyResponse {
+  ok: boolean;
+  memo?: string;
+  error?: string;
+}
+
+export async function revokeApiKey(req: Request, res: Response<RevokeApiKeyResponse>) {
```

Comment:
To eliminate the need for guesswork when working against these routes, I think it's good practice to create a type for the request. Even if it's not necessarily imported by the caller, it makes it obvious what needs to go where in the request. It also makes it easier within the function when everything isn't `any` :angel: